scripts/build_bytebase_docker.sh

```diff
@@ -18,6 +18,8 @@ docker build -f ./scripts/Dockerfile \
     --build-arg GIT_COMMIT="$(git rev-parse HEAD)"\
     --build-arg BUILD_TIME="$(date -u +"%Y-%m-%dT%H:%M:%SZ")"  \
     --build-arg BUILD_USER="$(id -u -n)" \
+    --progress=plain \
+    --network=host \
```

Comment:
Why should we add these arguments?

Reply:
only used for debugging. I will remove these.